ceph/ceph_admin/osd.py

```diff
@@ -106,3 +106,106 @@ def apply(self, config: Dict) -> None:
             sleep(interval)
 
         raise OSDServiceFailure("OSDs are not up and running in hosts")
+
+    def rm_status(self, config):
+        """
+        Execute the command ceph orch osd rm status.
+        Args:
+            config: command remove is passed from the test case.
+            op: operation parameters ex: remove
+        Example:
+            Testing ceph orch osd rm status
+        config:
+          command: rm
+
+        Returns:
+          output, error   returned by the command.
+
+        example
+            config:
+                command: rm status
+                base_cmd_args:
+                    verbose: true
+                args:
+                    format: json-pretty
+
+        """
+
+        base_cmd = ["ceph", "orch", "osd", "rm", "status"]
+        if config.get("base_cmd_args"):
+            base_cmd.append(config_dict_to_string(config["base_cmd_args"]))
+
+        if config and config.get("args"):
+            args = config.get("args")
+            base_cmd.append(config_dict_to_string(args))
+
+        return self.shell(args=base_cmd)
+
+    def rm(self, config):
+        """
```

Comment:
Since our API documentation used Rst format, please follow doc string format as mentioned in this method,
https://github.com/red-hat-storage/cephci/blob/master/ceph/ceph_admin/daemon.py#L13-L31

Reply:
Done